Dockerfile.i386

```diff
@@ -0,0 +1,30 @@
+FROM 32bit/debian
```

Comment:
What is the purpose of this image? Just for testing? Is that why you also install the Python bindings?